content/class/Profile.md

```diff
@@ -0,0 +1,206 @@
+| Object | [User](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/user) | [User](https://developer.twitter.com/en/docs/twitter-api/data-dictionary/object-model/user) | [Account](https://docs.joinmastodon.org/entities/account/) |
+
+## Properties
+
+### `description` {#property_description}
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public string description { get; }
+```
+
+How the user descripes itself.
+
+> This is the final version formatted for the use in the UI, formatted by [`format_text`](../../utils/namespace/ParseUtils.md#method_format_text), while the original text is found in [`raw_description`](#field_raw_description).
+
+### `created_at` {#property_created_at}
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public string created_at { get; }
+```
+
+When the User created his account.
+
+| API   | Twitter 1    | Twitter 2       | Mastodon        |
+| ----- | ------------ | --------------- | --------------- |
+| Field | `created_at` | `created_at`    | `created_at`    |
+| Type  | string       | ISO 8601 string | ISO 8601 string |
+
+### `fields` {#property_fields}
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public UserDataField[] fields { get; }
+```
+
+Fields providing additional informations about the User. See [`UserDataField`](../structure/UserDataField.md).
+
+| API   | Twitter 1          | Twitter 2          | Mastodon |
+| ----- | ------------------ | ------------------ | -------- |
+| Field | `location` & `url` | `location` & `url` | `fields` |
+| Type  | string             | string             | array    |
+
+### `header` {#property_header}
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2 (Not yet)](https://img.shields.io/badge/API-Twitter%202%20(Not%20yet)-darkred?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public Picture header { get; }
+```
+
+The header image of the User (for the profile page), stored as a [`Picture`](Picture.md).
+
+> Mastodon provides the option to use a animated GIF (which we store as a [`Animated`](Animated.md)) as a avatar. For now, we will not consider this option as it makes the code more complex and use always the static variant.
```

Comment:
This seems to be confusing "header image" (background) with avatar. Can Mastadon do animated headers, or is this comment against the wrong property? Or is it just a copy-paste because the header supports animation as well? (which sounds like a dreadful idea from the turn of the millennium with GeoCities sites!)

Reply:
Just a copy-paste issue.